src/components/common/button/CommonButton.tsx

```diff
@@ -12,7 +12,7 @@ const CommonButton = ({ text, href, hover, color }: buttonProps) => {
   return (
     <Link href={href}>
       <button
-        className={`btn transition-all py-4 px-10 rounded hover:bg-${hover} bg-${color} text-white`}
+        className={`btn transition-all py-4 px-10 rounded hover:${hover} ${color} text-white`}
```

Comment:
Does this break other places where common button was used?

Reply:
yes sir. Common Button colors didn't apply before the change. Now it's alright